src/Hl7.Fhir.Specification/Validation/Validator.cs

```diff
@@ -290,6 +293,9 @@ internal FhirPathCompiler FpCompiler
                     symbolTable.AddFhirExtensions();
 
                     _fpCompiler = new FhirPathCompiler(symbolTable);
+
```

Comment:
No - settings are read-only. In fact, the Settings property has a copy of the original settings passed to it, so it's no use setting it, no one will see it.